packages/twenty-front/src/modules/workflow/utils/assertUnreachable.ts

```diff
@@ -0,0 +1,3 @@
+export const assertUnreachable = (x: never, errorMessage?: string): never => {
```

Comment:
The function is inspired by https://stackoverflow.com/questions/39419170/how-do-i-check-that-a-switch-block-is-exhaustive-in-typescript.